tests/integration/check-custom-repo/test_custom-repos.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+@pytest.mark.good_tests
+def test_good_convertion(shell):
+    convertion = shell(
+        command=[
+            (
+                "convert2rhel -y "
+                "--serverurl {} --username {} "
+                "--password {} --pool {} "
+                "--enablerepo=rhel-server-rpms "
+                "--debug"
+            )
+            .format(
+                env.str("RHSM_SERVER_URL"),
+                env.str("RHSM_USERNAME"),
+                env.str("RHSM_PASSWORD"),
+                env.str("RHSM_POOL"),
+            )
+        ]
+    )
+    assert convertion.returncode == 0
+
+
+@pytest.mark.bad_tests
+def test_with_fake_repo_convertion(shell):
```

Comment:
I got:
```
            out: [05/04/2021 16:14:19] TASK - [Prepare: Checking if --enablerepo repositories are accessible] ****
            out: 
            out: Skipping the check of repositories due to the use of RHSM for the conversion.
            out: 
```
This is not expected, right?

Reply:
And it is inhibited, but not at the check level:
```
[05/04/2021 16:16:56] DEBUG - Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/convert2rhel/main.py", line 86, in main
    pre_ponr_conversion()
  File "/usr/lib/python3.6/site-packages/convert2rhel/main.py", line 189, in pre_ponr_conversion
    subscription.enable_repos(rhel_repoids)
  File "/usr/lib/python3.6/site-packages/convert2rhel/subscription.py", line 394, in enable_repos
    " subscription-manager:\n%s" % output)
  File "/usr/lib/python3.6/site-packages/convert2rhel/logger.py", line 103, in _critical
    sys.exit(msg)
SystemExit: Repos were not possible to enable through subscription-manager:
Error: 'fake-rhel-server-rpms' does not match a valid repository ID. Use "subscription-manager repos --list" to see valid repositories.
```

Reply:
Yes, it is expected since we are running with subscription manager.